kythe/release/appengine/buildbot/cloudbuild.yaml

```diff
@@ -0,0 +1,29 @@
+steps:
+- name: gcr.io/cloud-builders/gsutil
+  id: 'Fetch secrets'
+  args:
+  - cp
+  - gs://kythe-buildbot/secrets.tar.enc
+  - secrets.tar.enc
+- name: gcr.io/cloud-builders/gcloud
+  id: 'Decrypt secrets'
+  args:
+  - kms
+  - decrypt
+  - --location=global
+  - --keyring=Buildbot
+  - --key=secrets
+  - --ciphertext-file=secrets.tar.enc
+  - --plaintext-file=secrets.tar
+- name: gcr.io/cloud-builders/docker
+  id: 'Build image'
+  args:
+  - build
```

Comment:
does this do push automatically?

Reply:
Yes, because `images` was specified below, according to https://cloud.google.com/cloud-build/docs/configuring-builds/store-images-artifacts

Reply:
You can apparently do either or both, depending on needs.  I can add an explicit push, but it shouldn't be necessary.